src/app/feedback/feedback-form/feedback-form.component.ts

```diff
@@ -49,8 +50,9 @@ export class FeedbackFormComponent implements OnInit {
   }
 
   private getHeaderText(): string {
-    const el = this.el.element.nativeElement.querySelector('h1:not([style*="display:none"]');
-    return el ? el.innerHTML : '';
+    // const el = this.el.element.nativeElement.querySelector('h1:not([style*="display:none"]');
```

Comment:
Seems like it was working occasionally somehow (not sure how). Last one is after I threw away pseudoclasses and changed to `document.body`
<img width="698" alt="screen shot 2017-05-07 at 5 04 22 pm" src="https://cloud.githubusercontent.com/assets/20037396/25785153/a050335a-3347-11e7-8d1e-cda2c4177014.png">